lib/yggdrasil/player/registry.ex

```diff
@@ -0,0 +1,46 @@
+defmodule Yggdrasil.Player.Registry do
+  use GenServer
+  require IEx
+
+  @registry_ets :player_registry_ets
```

Comment:
Yeah, thinking about it a bit more this in specific to any game it's the global player record, I was thinking with the player registry as being game specific as well, but if we expand to multiple games the user id can actually be a token like

`game_id#user_id` which gets stored in the ets and would retrieve the user based on that unique key and there would relatively no need to even look up what game the process is for, because the player process knows what game it's talking to.

Sorry may be polluting these with stupid comments at this point, but I see what you did now and kind how you are thinking about it.